.github/workflows/build-ci.yml

```diff
@@ -78,3 +78,16 @@ jobs:
                   --platform ${{ matrix.platform }} \
                   --image ${{ env.image_tag }} \
                   --version ${{ matrix.version }}
+
+      - name: Run unit tests of entrypoint
```

Comment:
We don't have to run unit tests per image. I'm okay with keeping it for now, but we can also simply create a new workflow file and just run it, especially since we only have to trigger this test if the entrypoint code changes - integration testing will take care of all the other changes.

Reply:
I briefly tested using a new workflow file, launching tests from hosts. I add some permission issues with `chown` or `chmod`, I restored changes to current state to run it inside a container for now.

Reply:
Alright, let's pick it up in a separate change then. It's not blocking execution, just an optimization.